lib/vnet/ipc_credentials.go

```diff
@@ -33,13 +33,18 @@ import (
 )
 
 type ipcCredentials struct {
-	server, client credentials
+	// server holds credentials for the server-side of the connection (the VNet
+	// client application).
+	server credentials
+	// client holds credentials for the client-side of the connection (the VNet
+	// admin process).
+	client credentials
 }
 
 type credentials struct {
-	trustedCAPEM []byte
-	certPEM      []byte
-	signer       crypto.Signer
+	trustedCAPEM []byte        // X509 CA certificate
```

Comment:
nit: Make the comments match?

```suggestion
	trustedCAPEM []byte        // X.509 CA certificate
```